sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -0,0 +1,251 @@
+        """
+        return self._phone_number_client.phone_numbers.begin_release_phone_number(
+            phone_number,
+            **kwargs
+        )
+
+    @distributed_trace
+    def begin_search_available_phone_numbers(
+            self, 
+            country_code, 
+            phone_number_type, 
+            assignment_type, 
+            capabilities, 
+            area_code, 
+            quantity = 1,
+            **kwargs
+    ):
+        # type: (...) -> LROPoller[PhoneNumberSearchResult]
+        """Search for available phone numbers to purchase.
+
+        :param country_code: The ISO 3166-2 country code, e.g. US.
+        :type country_code: str
+        :param phone_number_type: Required. The type of phone numbers to search for, e.g. geographic,
+            or tollFree. Possible values include: "geographic", "tollFree".
+        :type phone_number_type: str or ~azure.communication.phonenumbers.models.PhoneNumberType
+        :param assignment_type: Required. The assignment type of the phone numbers to search for. A
+            phone number can be assigned to a person, or to an application. Possible values include:
+            "user", "application".
+        :type assignment_type: str or
+            ~azure.communication.phonenumbers.models.PhoneNumberAssignmentType
+        :param capabilities: Required. Capabilities of a phone number.
+        :type capabilities: ~azure.communication.phonenumbers.models.PhoneNumberCapabilities
+        :param area_code: The area code of the desired phone number, e.g. 425.
+        :type area_code: str
+        :param quantity: The quantity of phone numbers in the search. Should be at least 1.
+        :type quantity: int
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword polling: Pass in True if you'd like the LROBasePolling polling method,
+         False for no polling, or your own initialized polling object for a personal polling strategy.
+        :paramtype polling: bool or ~azure.core.polling.PollingMethod
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if no Retry-After header is present.
+        :rtype: ~azure.core.polling.LROPoller[~azure.communication.phonenumbers.models.PhoneNumberSearchResult]
+        """
+        search_request = PhoneNumberSearchRequest(
+            phone_number_type = phone_number_type,
+            assignment_type = assignment_type,
+            capabilities = capabilities,
+            area_code = area_code,
+            quantity = quantity
+        )
+        return self._phone_number_client.phone_numbers.begin_search_available_phone_numbers(
+            country_code,
+            search_request,
+            **kwargs
+        )
+    
+    @distributed_trace
+    def begin_update_phone_number_capabilities(
+            self, 
+            phone_number, 
+            sms, 
+            calling, 
```

Comment:
`sms` and `calling` are both required parameters here?
Is there any order in the parameters?